internal/controller/server/repository/job/job_test.go

```diff
@@ -76,7 +78,7 @@ func TestCheckJobIsAvailableForJobNotAvailable(t *testing.T) {
 	mockClient.On("GetValue", "metadata/testJobName").Return("", errors.New(constant.NoValueFound))
 
 	jobAvailabilityStatus, err := jobRepository.CheckJobIsAvailable(context.Background(), "testJobName")
-	assert.Equal(t, "job with testJobName name not found", err.Error())
+	assert.Equal(t, status.Error(codes.NotFound, errors.New(constant.Etcd+"job with testJobName name not found").Error()).Error(), err.Error())
```

Comment:
change errors.New(constant.Etcd+"job with testJobName name not found").Error() to constant.Etcd+"job with testJobName name not found"